pkg/kubernetes/database_cluster.go

```diff
@@ -29,27 +30,39 @@ import (
 
 // ListDatabaseClusters returns list of managed database clusters.
 func (k *Kubernetes) ListDatabaseClusters(ctx context.Context, namespace string) (*everestv1alpha1.DatabaseClusterList, error) {
-	return k.client.ListDatabaseClusters(ctx, namespace, metav1.ListOptions{})
+	result := &everestv1alpha1.DatabaseClusterList{}
```

Comment:
this is an example of the logic that replaces one from `pkg/kubernetes/client/` + `pkg/kubernetes/client/customresources` layers